README-ENGINES.md

```diff
@@ -1,8 +1,6 @@
-Engines
-=======
+# Engines
```

Comment:
Please revert this change (multiple locations). It was an explicit choice to use setext headings, not ATX headings, see commit message of 5f8e6c50bd862d6bb0bbd16b4c0b3be841d9cf32.

Reply:
Fixed on entire file